code/modules/surgery/organs/lungs.dm

```diff
@@ -556,8 +557,9 @@
 /obj/item/organ/internal/lungs/proc/too_much_tritium(mob/living/carbon/breather, datum/gas_mixture/breath, trit_pp, old_trit_pp)
 	var/gas_breathed = breathe_gas_volume(breath, /datum/gas/tritium)
 	// Tritium side-effects.
-	var/ratio = gas_breathed * 15
-	breather.adjustToxLoss(clamp(ratio, MIN_TOXIC_GAS_DAMAGE, MAX_TOXIC_GAS_DAMAGE))
+	if(gas_breathed > tritium_damage_moles_min)
```

Comment:
It should only do damage if the tritium overlay is visible on the tile

Reply:
`MOLES_GAS_VISIBLE 0.25` and `CELL_VOLUME 2500` meaning that you have 0.0001 moles in a litre when the gas is visible on a cell. 

Breath volume is 2 liters, so you want me to check that there is at least 0.0002 moles of tritium in there to deal damage, instead of my 0.01 moles right now, right?